src/controllers/auth.ts

```diff
@@ -46,6 +47,8 @@ export const loginGoogle: RequestHandler = (req, res, next) => {
 export const loginOneLogin: RequestHandler = (req, res, next) => {
     logger.debug('attempting to authenticate with one-login...');
 
+    const returnURL = `${config.frontend.url}/${req.language}/auth/callback`;
```

Comment:
Currently this will always be the browser language and not the language sent from the frontend (because it's been to the Auth provider and back before it hits this route).

I will fix this in an upcoming PR (using the session, a cookie, or some other method to remember the chosen language).